src/plugins/console/server/routes/api/console/mappings/register_get_route.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import type { IScopedClusterClient } from '@kbn/core/server';
+import type {
+  ClusterGetComponentTemplateResponse,
+  IndicesGetIndexTemplateResponse,
+  IndicesGetTemplateResponse,
+} from '@elastic/elasticsearch/lib/api/types';
+import { parse } from 'query-string';
+import type { RouteDependencies } from '../../..';
+import { API_BASE_PATH } from '../../../../../common/constants';
+
+interface Settings {
+  indices: boolean;
+  fields: boolean;
+  templates: boolean;
+  dataStreams: boolean;
+}
+
+async function retrieveSettings(
```

Comment:
having this method looks like extra proxy extra to me. You already defined `getMappings`, `getAliases`,  `getDataStreams`, `getTemplates`. Can we just put related logic directly in consumer methods?